### ISSUE TITLE
Is there really a need for multiple selection modes?

### ISSUE THREAD

**Issue by joewintergreen** [gabrielw-us/ProBuilder-Unreal-Public-Tracker#79]
I know this is How It's Always Been, but it's another thing I think is a dumb paradigm and slows everything down. If you just let everything be selectable at all times (or most of the time), what's lost? As long as your UI's set up decently, there's not a lot of potential for misclicks. There's very little ambiguity in any of the most common interactions.

![image](https://cloud.githubusercontent.com/assets/7104380/21539207/b09f5128-cdf8-11e6-8edb-a45b7365d2eb.png)

**Comment by gabrielw-us** [comment]
Yep, I've also been chatting with Karl about this on the Unity side. My thoughts are:

- have 3 toggle buttons: one for each element (vert, edge, face)
- All three can be on, or just two, or one
- When swapping between Element and Object editing, it "remembers" which elements to keep active, of course

I'd like this very much as well. What do you think, @arthursw ?

**Comment by joewintergreen** [comment]
Yeah, that'd be a good solution!

Gotta have those massive edge and vert lines on selected objects regardless, imo.

**Comment by arthursw** [comment]
Yes, agreed!
